data/project/ember/beta.md

```diff
@@ -5,14 +5,14 @@ filter:
   - /ember\./
   - /ember-template-compiler/
 repo: emberjs/ember.js
-initialVersion: 3.26.0 # Manually update, see https://libraries.io/npm/ember-source throughout
-lastRelease: 3.27.0-beta.4 # Manually update
-futureVersion: 3.27.0-beta.5 # Manually update
-finalVersion: 3.27.0 # Manually update
+initialVersion: 3.28.0-beta.1 # Manually update, see https://libraries.io/npm/ember-source throughout
+lastRelease: 3.28.0-beta.6 # Manually update
+futureVersion: 3.28.0-beta.7 # Manually update
+finalVersion: 3.28.0 # Manually update
 channel: beta
-cycleEstimatedFinishDate: 2021-05-03 12:00:00 # Manually update, change next one to 6 weeks from this date...regardless of delays in the release
-date: 2021-03-22 # Manually update, get date for `initialVersion`
-nextDate: 2021-05-03 12:00:00 # Manually update, change next one to 6 weeks from this date...regardless of delays in the release
+cycleEstimatedFinishDate: 2021-08-09 12:00:00 # Manually update, change next one to 6 weeks from this date...regardless of delays in the release
+date: 2021-05-03 # Manually update, get date for `initialVersion`
+nextDate: 2021-08-09 12:00:00 # Manually update, change next one to 6 weeks from this date...regardless of delays in the release
```

Comment:
I'm not clear on the difference between `cycleEstimatedFinishDate` and `nextDate`. Am I doing this correctly @ijlee2 @jaredgalanis @amyrlam ?

Reply:
@mixonic to be honest, I do not know also. Will add @locks to the chain in case he knows!

Reply:
The two fields are used in the `<Releases::ReleaseTimeline>` component (https://emberjs.com/releases/), as shown in the diagram below:

<img width="800" alt="difference between the dates" src="https://user-images.githubusercontent.com/16869656/125561241-819aa949-a3f3-4462-ae0d-f42e22994c41.png">

I'm not too sure what the component's author had had in mind, though, when they had come up with 2 different date fields.

Some of the fields in the Markdown files (for `release.md`, `beta.md`, and `canary.md`) are not used. I think, in a separate pull request, we could consider removing them for simplicity.